eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/JsonNodeToClusteredVariantProcessor.java

```diff
@@ -73,12 +78,21 @@ private ClusteredVariant parseJsonNodeToClusteredVariant(JsonNode jsonRootNode)
                 continue;
             }
             String assemblyAccession = assemblyInfo.get(0).path("assembly_accession").asText();
+            // Ignore variant if it's assembly accession doesn't match the one supplied in input parameters
+            if(!refseqAssembly.equals(assemblyAccession)) {
+                logger.error("Variant with RS ID {} has a different assembly accession {}"
+                        + "than the Reference Sequence accession {} supplied in input parameters",
```

Comment:
Again, to differentiate the service from a regular "reference sequence":
```suggestion
                        + "than the RefSeq accession {} supplied in input parameters",
```
It really annoys me they picked that name! But most people refer to them as 'RefSeq' anyway.